storey/timescaledb_target.py

```diff
@@ -112,6 +112,9 @@ def __init__(
         self._dsn = dsn
         self._pool = None  # Connection pool will be created lazily during first use
         self._column_names = self._get_column_names()
+        self._schema = None
+        if "." in self._table:
+            self._schema, self._table = self._table.split(".", 1)
```

Comment:
Should probably edit the docstring accordingly. Is the term schema correct here? I couldn't find anything about a `<schema>.<table>` form in TimescaleDB.